models/tokens/solana/tokens_solana_transfers.sql

```diff
@@ -0,0 +1,122 @@
+ {{
+  config(
+        schema = 'tokens',
+        alias = 'transfers',
+        materialized = 'incremental',
+        file_format = 'delta',
+        incremental_strategy = 'merge',
+        incremental_predicates = [incremental_predicate('DBT_INTERNAL_DEST.block_time')],
+        unique_key = ['tx_id','outer_instruction_index','inner_instruction_index'],
```

Comment:
If performance still isn't very good, adding block slot here could potentially help a bit (because it's sortable).

Also is it possible for either `outer_instruction_index` or `inner_instruction_index` to be `null`? If so this needs to be a surrogate key instead.